test/unit/org/apache/cassandra/utils/CassandraGenerators.java

```diff
@@ -99,7 +100,7 @@
 
     private static final Gen<SinglePartitionReadCommand> SINGLE_PARTITION_READ_COMMAND_GEN = gen(rnd -> {
         TableMetadata metadata = TABLE_METADATA_GEN.generate(rnd);
-        int nowInSec = (int) rnd.next(Constraint.between(1, Integer.MAX_VALUE));
+        long nowInSec = rnd.next(Constraint.between(1, Cell.MAX_DELETION_TIME));
```

Comment:
Why `Cell.MAX_DELETION_TIME`?

Reply:
Bc it's the max that makes sense? What would you suggest?

Reply:
That's what I'm asking, why that's the max that makes sense. That's a write time, and not a deletion time, isn't it?

Reply:
Our codebase will now be limited to 2186. So you want to test/use values up to that rather than only up to `Integer.MAX_VALUE` as `nowInSec` is now a `long` instead of an `int`. Am I answering your question?

Reply:
Yep, makes sense